components/dashboard/src/data/git-providers/unified-repositories-search-query.ts

```diff
@@ -62,7 +62,7 @@ export const useUnifiedRepositorySearch = ({
     }, [configurationSearch.data, excludeConfigurations]);
 
     const filteredRepos = useMemo(() => {
-        const repos = [suggestedQuery.data || [], searchQuery.data || [], flattenedConfigurations ?? []].flat();
+        const repos = [suggestedQuery.data || [], flattenedConfigurations ?? [], searchQuery.data || []].flat();
```

Comment:
This is done to artificially boost the priority of imported repos vs SCM-provided results.

Every time I change this, I think about how much we need to get rid of it entirely and just let `listSuggestedRepositories` receive a search string, which will do all deduplication and ordering by itself.